app/views/layouts/component_preview.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html style="background-color: <%= params.dig(:lookbook, :display, :bg_color) || 'white' %>">
+  <head>
+    <title>
+      Component Previews
+    </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= stylesheet_link_tag 'application_stylesheet', media: 'all' %>
+    <%= stylesheet_pack_tag 'application' %>
+    <%= javascript_pack_tag 'application', 'data-turbo-track': 'reload', defer: true %>
+  </head>
+  <body style="padding: 20px">
+    <div style="
+    margin-left: auto;
+    margin-right: auto;
+    max-width: <%= params.dig(:lookbook, :display, :max_width) || '100%' %>
+  ">
+      <%= yield %>
+      <%= render 'shared/ga' %>
```

Comment:
Nit: Is GA tracking needed?

Reply:
This one was specifically for the lesson completion button, which has a GA event hooked into it. I think I need to give that button access to a current user in some sort of application controller / before hook anyway, I'll try and think of a cleaner way to make the button happy